payjoin/src/receive/error.rs

```diff
@@ -86,6 +86,13 @@ pub(crate) enum InternalRequestError {
     ParsePsbt(bitcoin::psbt::PsbtParseError),
     #[cfg(feature = "v2")]
     Utf8(std::string::FromUtf8Error),
+    /// Original PSBT fee rate is below minimum fee rate set by the receiver.
+    ///
+    /// First argument is the calculated fee rate of the original PSBT.
+    /// [`UncheckedProposal::psbt_fee_rate`]
```

Comment:
remember to remove this comment if choosing to remove psbt_fee_rate from the public interface.